### ISSUE TITLE
Is user interaction needed to make this extension work?

### ISSUE THREAD

**Issue by sobitthen** [WorldLanguages/DoNotDiscard#8]
I loaded the extension, visited the Discards page, but unsure if we are supposed to make any changes or if this extension no longer works?
Commented on another open issue but it did not show an update on my end thus creating this issue.

Trying to stop 1 page from auto refreshing, Facebook, leave to research something I had seen there, come back to make informed comment, and the page refreshes and the post is lost... 
Do we need to interact in some manner on the Discards page or should this extension be working without user interaction?
Likely some update to Chrome as 3 5 star reviews on Chrome store claiming it is working, no joy here...

**Comment by WorldLanguages** [comment]
> Is user interaction needed to make this extension work?

No, it's not needed. If the extension is installed it should work. If you installed other extensions that change the autoDiscardable property of tabs, disable them.

Multiple Chrome updates have changed the tab discarding algorithms and functionality over the years. Setting tabs as not auto-discardable doesn't always prevent the browser from silently killing tabs if there's not enough memory in your device.
